tests/pq/test_pq.py

```diff
@@ -1,4 +1,5 @@
 import os
+import sys  # noqa: F401  # used in skipif
```

Comment:
Can't we a boolean instead of a "condition string" in this `skipif` so as to avoid this `# noqa` here?

Reply:
Yes, but then we need to specify a reason, whereas, with a string, pyteat will do it for you.

But in this case I added a sort of reason as common, so yes,you are right: better as a bool here :)